webextensions/manifest/background.json

```diff
@@ -58,8 +58,7 @@
                 "notes": "Available for use in Manifest V2 only."
               },
               "edge": {
-                "version_added": "14",
-                "notes": "This property is required."
+                "version_added": "14"
```

Comment:
@queengooborg this should simply "mirror" chrome now.

Reply:
The property is actually still required below EdgeHTML v45 before Microsoft edge switched to Chromium.

Reply:
@queengooborg Wonderful! Thanks